### ISSUE TITLE
Design Logo

### ISSUE THREAD

**Issue by ishitakapoor26** [smaranjitghose/DocLense#283]
Design Logo for DocLense

**Comment by ishitakapoor26** [comment]
pls assign me this issue.

**Comment by ramanjitsingh-hub** [comment]
can i contribute to this issue

**Comment by ramanjitsingh-hub** [comment]
I have a logo design 
should I change it in the codebase and raise a PR or just submit it here

**Comment by anushbhatia** [comment]
can you send the sample here before creating a PR for that.

**Comment by ramanjitsingh-hub** [comment]
Two of the designs I have selected. Suggestions will be appreciated

![DocLense (1)](https://user-images.githubusercontent.com/63701517/149712298-bbb9be33-3a8a-4bd4-a671-68b1179b9cd4.png)

![DocLense](https://user-images.githubusercontent.com/63701517/149712381-7f692131-91b7-4bf7-b870-5f04b0b6621d.png)

**Comment by smaranjitghose** [comment]
@ramanjitsingh-hub,
Although @anushbhatia is the expert on the design side and his verdict is final, I must say your designs are clean and crisp and conform to our theme. Would it be possible for you to give an Indian touch to lens or maybe the text below?

**Comment by ramanjitsingh-hub** [comment]
Yeah I can do that. Can you confirm which of the two above should I change.If you want to review both of the logos with some Indian accent I can do that too

**Comment by anushbhatia** [comment]
the 2nd one refine the edges on the inner border cause the strokes are a bit outside it.

**Comment by ramanjitsingh-hub** [comment]
How about this one
![DocLense (3)](https://user-images.githubusercontent.com/63701517/149918813-fd4df491-ae7e-4786-933e-8815cbd07e80.png)

**Comment by smaranjitghose** [comment]
Awesome! @anushbhatia your comments?

**Comment by anushbhatia** [comment]
Some minor changes
1. Avoid using black on the borders 
2. You can improve the colours of blue
3. doclense might be read as docolense beacuse of ashok chakra
